examples/specs/window_transform_time_spent_in_class.vl.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+    "description": "Bar graph showing how each film differs from the average rating for that year",
```

Comment:
outdated description?

Reply:
I'm also not sure what this plot try to communicate.

Reply:
I was trying to make a more complicated window transform example. The purpose was to show how long each student has been in class until the current class. So if student A and B both show up to a class, the graph will show that class along with the total time each student has been at school, to show how tired (or the fatigue) of a student during that class. For example, something like this could be used to see the affects of taking an exam after a lot of other classes vs taking an exam at the beginning of the day or when there are a few classes before hand.

I changed the name of the axis to be `Time Spent in Class` instead to hopefully make this more clear. I have also updated the description.